bedrock/thunderbird/templates/thunderbird/start/release.html

```diff
@@ -34,6 +72,7 @@ <h2 class="iconic">{{ _('Need Support?') }}</h2>
           <ul>
             <li><a href="{{ url('thunderbird.features') }}">{{ _('Features') }}</a></li>
             <li><a href="https://blog.mozilla.org/thunderbird/">{{ _('Visit our blog') }}</a></li>
+            <li><a href="https://donate.mozilla.org/thunderbird/about/">{{ _('Donate') }}</a></li>
```

Comment:
Is it intentional that this donate link goes to `/thunderbird/about/`, whereas the other one just goes to `/thunderbird/`? Both seem to have a donate form, but just wanted to make sure they should not in fact be the same link 😄